ethereum/core/src/main/java/tech/pegasys/teku/core/epoch/status/ValidatorStatuses.java

```diff
@@ -135,4 +137,53 @@ public void apply(final ValidatorStatus status) {
       status.updateInclusionInfo(inclusionInfo);
     }
   }
+
+  public static TotalBalances createTotalBalances(final List<ValidatorStatus> statuses) {
```

Comment:
nit: This probably should just be private - you'd just create `ValidatorStatuses` and then use `getTotalBalances` to get it.

Reply:
Done.